docs/operating-scylla/security/security-checklist.rst

```diff
@@ -31,7 +31,9 @@ Encryption on Transit, Client to Node and Node to Node
 Encryption on Transit protects your communication against a 3rd interception on the network connection.
 Configure Scylla to use TLS/SSL for all the connections. Use TLS/SSL to encrypt communication between Scylla nodes and client applications.
 
-See:
+.. only:: enterprise
+
+    ScyllaDB Enterprise can run on FIPS-compliant systems since version 2023.1.1.
```

Comment:
```suggestion
Starting with ScyllaDB Enterprise 2023.1.1, ScyllaDB can run on FIPS-enabled Ubuntu, using libraries that were compiled with FIPS enabled, like OpenSSL, GnuTLS, and more. 
```

Reply:
I think we should stick to "FIPS-compliant" (as we're not talking about Ubuntu containers here). Also, the explanation seems too technical (referring to compilation). Assuming it's only Ubuntu, we could say:

_Starting with version 2023.1.1, you can run ScyllaDB Enterprise on FIPS-compliant Ubuntu, using FIPS 140-2 certified libraries, such as OpenSSL, GnuTLS, and more._ 

@tzach Are you OK with the above?